app.json

```diff
@@ -0,0 +1,24 @@
+{
+  "cron": [
+    {
+      "command": "python -m metrics github pr-throughput ebmdatalab $(date -d '1 day ago' '+%F')",
+      "schedule": "@daily"
+    },
+    {
+      "command": "python -m metrics github pr-throughput opensafely-core $(date -d '1 day ago' '+%F')",
+      "schedule": "@daily"
+    },
+    {
+      "command": "python -m metrics slack tech-support $(date -d '1 day ago' '+%F')",
+      "schedule": "@daily"
+    },
+    {
+      "command": "python -m metrics github open-prs ebmdatalab $(date -d '+%F')",
```

Comment:
The command `date -d '+%F'` gives the error: `date: invalid date ‘+%F’` when I run it locally because -d expects an argument (copy and paste error?).

Reply:
arg, yep, thanks!  Luckily all the "yesterday" bits are now going away thanks to #51